packages/template-retail-react-app/app/pages/order-status/index.jsx

```diff
@@ -39,8 +46,11 @@ const OrderStatusPage = () => {
     return (
         <Box data-testid="order-status-page" bg="gray.50">
             <Container py={{base: 8, md: 8}} pt={{base: 12, md: 24}}>
-                <Heading as="h1" size="lg" textAlign="left">
-                    Order Status
+                <Heading as="h1" size="lg" textAlign="left" tabIndex="0" ref={headingRef}>
```

Comment:
```suggestion
                <Heading as="h1" size="lg" textAlign="left" tabIndex="-1" ref={headingRef}>
```
So that it's not keyboard tab accessible.